sw/device/silicon_creator/rom/sigverify_keys_ecdsa_p256.c

```diff
@@ -0,0 +1,37 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/rom/sigverify_keys_ecdsa_p256.h"
+
+#include "sw/device/silicon_creator/rom/sigverify_otp_keys.h"
+
+#include "otp_ctrl_regs.h"
+
+rom_error_t sigverify_ecdsa_p256_key_get(const sigverify_otp_key_ctx_t *sigverify_ctx,
+                                  uint32_t key_id, lifecycle_state_t lc_state,
+                                  const sigverify_ecdsa_p256_buffer_t **key) {
+  *key = NULL;
+  rom_error_t error = kErrorSigverifyBadSpxKey;
```

Comment:
Should this be: `kErrorSigverifyBadEcdsaKey` ?

Same below?

Reply:
Fixed.